com.github.ransome1.sleek.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=0.4.1
+Name=com.github.ransome1.sleek
+Exec=/app/bin/run.sh %u
+Type=Application
+Icon=/app/share/applications/com.github.ransome1.sleek.png
```

Comment:
`Icon` should just be the icon name, without extension. That's why the build is failing.